tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/matterdashboard.controller.js

```diff
@@ -51,6 +51,13 @@
             vm.selectedTabInfo = vm.matterDashboardConfigs.Tab2HeaderText + " (" + vm.allMatterCount + ")";
             vm.Pinnedobj = [];
             //#endregion            
+            vm.teamName = '';
+            //#region Get Querystring values
+            if ($location.search() && $location.search().teamName) {

```

Comment:
Code changes for filtering the matters based on the client name on the matter landing page